### PR TITLE
fixed type error with demisto args passed in

### PR DESCRIPTION
**Pull request by clay584**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
When pulling relevant logs from Logz.io related to an event ID, the integration fails due to a type error with the args provided to the command/integration. This fix just coerces the args to `int` to avoid the python exception `Failed to execute command. Error: '>' not supported between instances of 'str' and 'int'`

There is an open Logz.io support case for this. It is 23168 for your reference.

### CODE REVIEW

**Review by yyyogev — APPROVED** [review]
Looks good on my behalf
Thanks

**Review by esharf — APPROVED** [review]

**Comment by esharf** [comment]
@clay584 please fix the conflicts so we will be able to merge

**Comment by esharf** [comment]
@clay584 you are overriding the current RN but you should add a new one and bump the pack version (in `pack-netadta.json` file).
you can do it with `demisto-sdk` (more info [here](https://xsoar.pan.dev/docs/integrations/release-notes)).
please add the new and we are free to merge.

File: Packs/Logzio/ReleaseNotes/1_1_2.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 
 #### Integrations
 ##### Logz.io
-- Upgraded the Docker image to demisto/python3:3.8.6.13358.
\ No newline at end of file
+- Upgraded the Docker image to demisto/python3:3.8.6.13358.
```
Comment (anara123) [inline Packs/Logzio/ReleaseNotes/1_1_2.md:4]: undo this change.

Comment (anara123) [reply Packs/Logzio/ReleaseNotes/1_1_2.md:4]: Make sure this file don't appears in the changed files

File: Packs/Logzio/ReleaseNotes/1_1_3.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Logz.io
+- Fixed integration argument bug when fetching event logs from Logz.io.
+- Upgraded the Docker image to demisto/python3:3.8.6.13358.
```
Comment (anara123) [inline Packs/Logzio/ReleaseNotes/1_1_3.md:5]: delete this line. it belongs to the previous release notes

**Comment by bakatzir** [comment]
We will handle the conflicts ourselves.
